### PR TITLE
Added Result Page

### PR DESCRIPTION
**Pull request by Ankur9669**
image

### CODE REVIEW

File: quizanswers.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.quiz-question-answer-container{
+    margin: 0rem 2rem;
+}
+.quiz-questions-container{
+    margin-block-start: 0px;
+    margin-block-end: 0px;
+    margin: 0rem 1.7rem;
+    padding-inline-start: 0px;
```
Comment (sri-rishi) [inline quizanswers.css:8]: Suggestion: using 0 instead of 0rem or 0px will work fine

**Review by sri-rishi — CHANGES_REQUESTED** [review]
Great work on UI and Code, just some changes can be done which I have commented

File: quizanswers.css (hunk 1 above)
Comment (Ankur9669) [reply quizanswers.css:8]: Sarre PR par yehi gadbad karta mai😂

**Review by manish-GH — COMMENTED** [review]
The code looks fine and the UI is good, but the page looks kind of cluttered. You could probably add some spacing or some kind of a separator for each question. (Or try adding some left padding for the options )

File: quizanswers.css (hunk 1 above)
Comment (Ankur9669) [reply quizanswers.css:8]: Done

**Review by Sahil25061999 — APPROVED** [review]
The code and ui looks beautiful, the retry icon can be different ( just a personal opinion ), add spacing between the icon and   text for the retry button.

**Review by sri-rishi — APPROVED** [review]

**Pull request merged**